Issue#923 Update VAT configuration page (#1274)
magento/merchdocs: src/tax/vat-configure.md

## Step 1: Set up customer tax classes

The process of creating a tax rule begins by adding a tax rate.

1. On the _Admin_ sidebar, go to **Stores** > _Taxes_ > **Tax Zones and Rates**.

1. Ensure that there is a customer tax class that is appropriate to use with the VAT.

   For this example, ensure that there is a customer tax class named Retail Customer. If Retail Customer does not exist, click <span class="btn">Add New Tax Rate</span>.

    ![]({% link images/images/configuring-vat-tax-zones-and-rates.png %}){: .zoom}
    _Set up customer tax classes_

> 1. Enter the **Tax Identifier** for the new tax class. All tax rates displayed in the **Tax Rate** field of the **Tax Rule Information**, for each tax rules.

Reviewer: ```suggestion
1. Enter the **Tax Identifier** for the new tax class.

   All tax rates are displayed in the _Tax Rate_ field in the _Tax Rule Information_ when you create tax rules.
```
